app/models/ingredient.rb

```diff
@@ -0,0 +1,13 @@
+class Ingredient < ApplicationRecord
+
+  has_many :dish_ingredients, dependent: :destroy
+  has_many :dishes, through: :dish_ingredients
+
+  def find_dish_ingredient(dish_id)
+    DishIngredient
```

Comment:
See comment in controller about using find_by
`dish_ingredient = DishIngredient.find_by(dish_id: dish_id, ingredient_id: ingredient_id)`